man/systemd.network.xml

```diff
@@ -193,6 +193,15 @@
           This happens when multicast routing is enabled.</para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>IPv6AddressGenMode=</varname></term>
```

Comment:
hmm, I'd prefer if we wouldn't abbreviate this needlessly, i.e. call this IPv6AddressGenerationMode= please. Or given this is for link-local stuff only (right?) maybe even IPv6LinkLocalAddressGenerationMode=